terraform/shared_account_pathtolive_infra_ci/e2e_tests.tf

```diff
@@ -40,6 +40,10 @@ module "run_e2e_tests" {
     {
       name  = "STACK_TYPE"
       value = "next"
+    },
+    {
+      name  = "CODEPIPELINE_EXECUTION_ID"
+      value = "#{codepipeline.PipelineExecutionId}"
```

Comment:
Is the # here a typo? I've not seen that syntax before

Reply:
This is a reference to an exported Codepipeline variable

Reply:
It's a template variable : https://docs.aws.amazon.com/codepipeline/latest/userguide/reference-variables.html#reference-variables-list